docs/design/coreclr/botr/guide-for-porting.md

```diff
@@ -0,0 +1,440 @@
+---
+Guide for porting .NET to a new processor architecture
+---
+
+This document is broken up into 2 major sections.
+
+1.  The various porting stages of porting the .NET Runtime
+
+2.  A technical discussion of the major components affected by a port to a new
+    architecture
+
+Porting stages and steps
+========================
+
+Porting the .NET Runtime to a new architecture typically follows along the
+following path.
+
+As engineering continues along the development path, it is best if the logic can
+be placed into the master repository of the runtime as soon as possible. This
+will have 2 major effects.
+
+1.  Individual commits are easier to review
+
+2.  Not all approaches for fixing problems will always be considered acceptable.
+    It is plausible that a some change may not ever be acceptable to take into
+    the public git repo, and discovering such issues early can avoid large
```

Comment:
```suggestion
    the upstream git repo, and discovering such issues early can avoid large
```

Anybody can have public fork with anything they wish. We do not have control over that. upstream is what matters.